.github/workflows/push-to-testnet-draft.yml

```diff
@@ -15,20 +15,28 @@ jobs:
         uses: actions/checkout@v2
       - name: Random sleep (balena API workaround)
         run:  sleep $[ ( $RANDOM % 180 )  + 1 ]s
-      - name: Change i2c bus for RockPi
+      - name: Generate Docker Compose for RockPi
```

Comment:
Is this file intentional?

Reply:
i created this a while ago so we could manually push draft releases to balena using workflow-dispatch